src/library/Pagination/PageJumper.js

```diff
@@ -0,0 +1,70 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { createStyledComponent } from '../styles';
+import { FlexItem } from '../Flex';
+import { FormField } from '../Form';
+import TextInput from '../TextInput';
+import type { Messages } from './Pagination';
+
+type Props = {
+  'aria-label'?: string,
+  currentPage: number,
+  inputRef: (node: HTMLInputElement) => HTMLInputElement,
```

Comment:
Methinks that the TextInput component's innerRef flow type, and that within other components should most likely be defined with the DOM element as the node rather than React$Component.  We would of course need to verify that all of the refs were actually applied this way.  Regardless, try these updates in your `PageJumper`.  They seem to work for me locally.

```js
pageJumperInput: ?HTMLInputElement;
...
inputRef: this.setPageJumperRef,
...
setPageJumperRef = (node: ?HTMLInputElement) => {
  this.pageJumperInput = node;
};
```

Reply:
Oh, I just pushed the udpate...